### ISSUE TITLE
Ubuntu Kinetic builds break

### ISSUE THREAD

**Issue by totaam** [Xpra-org/xpra#3609]
They've changed `debuild`:
```
dpkg-buildpackage: error: unknown option or argument --no-lintian
```
Then later:
```
/usr/bin/ld: cannot find -lnvidia-fbc: No such file or directory
```
Despite:
```shell
$ ls -la opt/Capture_Linux/NvFBC/libnvidia-fbc.so*
lrwxrwxrwx. 1 buildbot buildbot     18 Jun  2  2021 opt/Capture_Linux/NvFBC/libnvidia-fbc.so -> libnvidia-fbc.so.1
lrwxrwxrwx. 1 buildbot buildbot     23 Jun  2  2021 opt/Capture_Linux/NvFBC/libnvidia-fbc.so.1 -> libnvidia-fbc.so.465.31
-rwxr-xr-x. 1 buildbot buildbot 129760 Jun  2  2021 opt/Capture_Linux/NvFBC/libnvidia-fbc.so.465.31
```
And https://github.com/Xpra-org/repo-build-scripts/blob/9fdc27b6e4738228cab3aca18709a1485c12fd94/pkgconfig/nvfbc.pc#L1-L8
```
prefix=/opt/Capture_Linux/NvFBC
(..)
Libs: -L${prefix} -lnvidia-fbc
```

**Comment by totaam** [comment]
More splatter:
```
Unpacking systemd-resolved (251.4-1ubuntu1) ...
dpkg: error processing archive /var/cache/apt/archives/systemd-resolved_251.4-1ubuntu1_amd64.deb (--unpack):
 unable to make backup link of './etc/resolv.conf' before installing new version: Invalid cross-device link
```
And after that, nothing can be installed properly..

**Comment by totaam** [comment]
Seems like this has been fixed and we now have beta builds for Kinetic.
```
-rw-r--r--. 1 root root  3149600 Sep  3 16:02 ../xpra_4.4-r31782-1_amd64.deb
```
